README.md

```diff
@@ -1,6 +1,58 @@
 # covid19-india
 
+## Repo Structure
+    The repository is structured as follows:
+    ./
+    |-- config    : Various config params to be entered by humans go here
+    |-- data      : All data go in here, files are tracked by git-lfs
```

Comment:
Is "data" input data ? should we clarify since next line says output data. An example might help as well

Reply:
Sure, will name it input and output.